.github/workflows/main.yml

```diff
@@ -6,6 +6,10 @@ on:
 
 jobs:
   build:
+    env:
+      CI: true
```

Comment:
This can be removed. It's already set in GitHub Actions by default.

https://docs.github.com/en/actions/learn-github-actions/environment-variables#default-environment-variables

Reply:
done, thanks